erdgo/utils.go

```diff
@@ -7,8 +7,16 @@ import (
 	"github.com/ElrondNetwork/elrond-go/crypto/signing"
 	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519"
 	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519/singlesig"
+	"github.com/ElrondNetwork/elrond-go/hashing/keccak"
 )
 
+const (
```

Comment:
maybe you can add in cmd/testTransactionSignedWithHash

Reply:
Done